README.md

```diff
@@ -1,104 +1,159 @@
-- [gis](https://github.com/tarantool/gis): store and query geospatial data
-- [gperftools](https://github.com/tarantool/gperftools): collect CPU profile to find bottlenecks in your code
+- [avro-schema](https://github.com/tarantool/avro-schema): Apache Avro scheme for your data.
+- [connpool](https://github.com/tarantool/connpool): Keep a pool of connections to other Tarantool instances.
+- [curl](https://github.com/tarantool/curl): HTTP client based on libcurl.
+- [expirationd](https://github.com/tarantool/expirationd): Automatically delete tuples based on expiration time.
+- [gis](https://github.com/tarantool/gis): Store and query geospatial data.
+- [gperftools](https://github.com/tarantool/gperftools): Collect CPU profile to find bottlenecks in your code.
+- [http](https://github.com/tarantool/http): Embedded HTTP server with flask-style routing support.
+- [memcached](https://github.com/tarantool/memcached): Access Tarantool as if it was a Memcached instance.
+- [metrics](https://github.com/tarantool/metrics): Metric collection library for Tarantool.
+- [mqtt](https://github.com/tarantool/mqtt): Client for MQTT message brokers.
+- [mysql](https://github.com/tarantool/mysql): Query MySQL right from Tarantool.
+- [pg](https://github.com/tarantool/pg): Query PostgreSQL right from Tarantool.
+- [prometheus](https://github.com/tarantool/prometheus): Instrument code and export metrics to Prometheus monitoring.
+- [queue](https://github.com/tarantool/queue): Priority queues with TTL and confirmations.
+- [vshard](https://github.com/tarantool/vshard): Automatically distribute data across multiple instances.
 
-If the module you need is not listed here, there is a good chance we may add it. Open an issue [on our GitHub](https://github.com/tarantool/docker).
 
-# Data directories
+If the module you need is not listed here, there is a good chance we can add it.
+Open an issue [on our GitHub](https://github.com/tarantool/docker).
 
-- `/var/lib/tarantool` is a volume containing operational data
-  (snapshots, xlogs and vinyl runs)
+### Data directories
 
-- `/opt/tarantool` is a place where users should put their lua
-  application code
+Mount a volume to these directories to make use of them.
 
-# Convenience utilities
+- `/var/lib/tarantool` contains operational data
+  (snapshots, xlogs and vinyl runs).
 
-- `console`: execute it without any arguments to open administrative
-  console to a running Tarantool instance
+- `/opt/tarantool` is the directory for Lua application code.
 
-- `tarantool_is_up`: returns 0 if Tarantool has been initialized and
-  is operating normally
+### Convenience tools
+
+- `console`: execute it without arguments to open an administrative
+  console to a running Tarantool instance.
+
+- `tarantool_is_up`: returns `0` if Tarantool has been initialized and
+  is operating normally.
 
 - `tarantool_set_config.lua`: allows you to dynamically change certain
   settings without the need to recreate containers.
```

Comment:
```suggestion
- `console`: Execute without arguments to open an administrative
  console to a running Tarantool instance.

- `tarantool_is_up`: Returns `0` if Tarantool has been initialized and
  is operating normally.

- `tarantool_set_config.lua`: Allows you to dynamically change certain
  settings without the need to recreate containers.
```
Use consistent capitalization after colons in all sections. I corrected this snippet according to what I saw above.